### ISSUE TITLE
Enable the feedback link in the footer to have different title and aria-label content

### ISSUE THREAD

**Issue by KPobeeNorris** [citizensadvice/design-system#3509]
# Footer Component

At present you are able to pass in a feedback_title argument to the FooterComponent, and if it's included the link title and aria-label will use the feedback_title content e.g. the following screenshot is where the feedback_title is passed in as `Is there anything wrong with this page? Let us know`

![Screenshot 2024-10-15 at 08 51 42](https://github.com/user-attachments/assets/74e46d3d-cd6b-4df5-b783-bcc35088106c)

However, we were recently asked whether it was possible to set the aria-label to `Is there anything wrong with this page? Let us know by completing a feedback form. (opens in a new tab)`, but this isn't possible at present as the footer component only takes the feedback_title arg.  Would it therefore be possible to add another optional `aria-label` argument to the component, and if it's not present then fallback to the feedback title?

**Comment by davidrapson** [comment]
Is this the same as https://github.com/citizensadvice/design-system/pull/3517 @marianayovcheva?

**Comment by marianayovcheva** [comment]
I think it's different - #3517 was fixing a bug because we didn't have a fallback for the `aria-label` title. This issue is about making `aria-label` customisable and separating it from `feedback_link.title`. I'd imagine the priority of the aria-label text to be:
- `some custom aria label (opens in a new tab)` (new functionality)
- `feedback link title (opens in a new tab)`
- `Is there anything wrong with this page? Let us know (opens in a new tab)` (our default text)

In the example with `Is there anything wrong with this page? Let us know by completing a feedback form. (opens in a new tab)`, I wonder if it might be worth it displaying `by completing a feedback form` to all users instead 🤔.

**Comment by KPobeeNorris** [comment]
Yeah, this is a different issue from the one @marianayovcheva has kindly fixed 🙂 As per her comment it was about being able to ensure the aria-label was as descriptive and as clear as possible, which might be slightly different to the feedback title.  I mentioned about passing in an optional argument as I didn't know how widely used this component is, and therefore what the impact of changing the default text would be.  However, if it's easier, and wouldn't impact anyone negatively, to add  Mariana's suggestion of adding `by completing a feedback form` then that also works for me 😄
